shelley/chain-and-ledger/executable-spec/src/LedgerState.hs

```diff
@@ -1163,9 +1163,9 @@ overlaySchedule e gkeys pp = do
         toRelativeSlotNo x = (Duration . floor) (dInv * fromInteger x)
         toSlotNo x = firstSlotNo +* toRelativeSlotNo x
 
-        genesisSlots = [ toSlotNo x | x <-[0..(floor numActive)] ]
+        genesisSlots = [ toSlotNo x | x <-[0..(floor numActive - 1)] ]
 
-        numInactivePerActive = floor ((1 - asc) * fromRational numActive) - 1
+        numInactivePerActive = floor (1 / asc) - 1
```

Comment:
I am not sure I agree, e.g., for `f=0.9` we'd have 0 inactive slots this way, wouldn't we want 9 active ones and then an inactive slot ?

Reply:
but it's probably ok as a real active slot coefficient would be way below `0.5`, which is the threshold for the above effect, right?

Reply:
good point about `f>=5`.